tests/unit/routes/settings/billing-info-test.js

```diff
@@ -0,0 +1,11 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Route | settings/billing-info', function(hooks) {
+  setupTest(hooks);
+
```

Comment:
Why additional router tests, You could have included the testing of the following in the `Acceptance/Settings Test`. And since you will be adding components to the following routes, So this router test I don't think are necessary, Once you have added things in your templates, You may acceptance tests for the following.
@shreyanshdwivedi @uds5501 Please drop your views as well

Reply:
@kushthedude I agree. It's an auto-generated test. It was created when I used ember-cli. Let's wait for what others want to say.

Reply:
@mrsaicharan1 I agree with @kushthedude here. We can have acceptance tests in `Acceptance/Settings` itself

Reply:
Shifted the tests. :D